app/docs/enterprise/0.31-x/getting-started/adding-your-api.md

```diff
@@ -10,13 +10,16 @@ step to having Kong EE manage your API. For purposes of this Getting Started
 guide, we suggest adding the [Mockbin API][mockbin] to Kong, as Mockbin is
 helpful for learning how Kong EE proxies your API requests.
 
-Kong exposes a [RESTful Admin API][API] on port `:8001` for managing the
-configuration of your Kong instance or cluster.
+Kong exposes a [RESTful Admin API][API] on ports `:8001` and `:8444` and an
+Admin GUI on ports `:8002` and `:8445` for managing the
+configuration of your Kong instance or cluster. The Admin GUI simply makes
+requests to the Admin API, so you can use either interface for configuring
+and managing Kong EE.
 
-### 1. Add your API using the Admin API
+## 1. Add your API using the Admin API or GUI
 
-Issue the following cURL request to add your first API ([Mockbin][mockbin])
-to Kong:
+If you'd like to use the Admin API, issue the following cURL request to add 
```

Comment:
capitalize the c in CURL

Reply:
Wikipedia suggests not https://en.wikipedia.org/wiki/CURL

Reply:
huh TIL

> The name originally stood for "see URL"